### PR TITLE
Add the demo scripts

### PR DESCRIPTION
**Pull request by kerneltime**
To see the demos head to https://github.com/vmware/docker-volume-vsphere/wiki/Demos

### CODE REVIEW

**Comment by kerneltime** [comment]
The ESX admin monitoring demo was done manually as the default width in the automated demos does not work with the admin CLI and currently i am battling the tools to figure out the automated solution. For now there is only a manual option for the admin CLI.

File: misc/demos/docker-volume-commands.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+title: Run docker volume commands
+scenes:
+  - name: Docker volume commands
+    action: echo "Post install, use the plugin for docker volume operations"
+  - name: Create a volume
+    action:  docker volume create --driver=vmdk --name=MyVolume -o size=10gb
```
Comment (pdhamdhere) [inline misc/demos/docker-volume-commands.yaml]: Formatting 10gb would be slow? Make it  1GB?

File: misc/demos/install-vm.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    action: echo "Download RPM or Deb from GitHub Releases. Using RPM on Photon here.."
+  - name: Curl rpm
+    action: curl -OL https://github.com/vmware/docker-volume-vsphere/releases/download/1.0.beta/docker-volume-vsphere-1.0.beta-1.x86_64.rpm
+    wait: true
+  - name: just a pause
+    pause: 1
+  - name: Copy RPM
+    action: scp docker-volume-vsphere-1.0.beta-1.x86_64.rpm root@$VM:/tmp
+    wait: true
+  - name: Install RPM
```
Comment (pdhamdhere) [inline misc/demos/install-vm.yaml]: Add new lines between major commands?

**Comment by pdhamdhere** [comment]
Nice. Like it! LGTM.

File: misc/demos/docker-volume-commands.yaml (hunk 1 above)
Comment (kerneltime) [inline misc/demos/docker-volume-commands.yaml]: The demo seems fine..

File: misc/demos/install-vm.yaml (hunk 2 above)
Comment (kerneltime) [inline misc/demos/install-vm.yaml]: Ok.

**Comment by msterin** [comment]
Veeeery nice. 2 small comments - it would be nice to have '#' before comment lines in shell (when you give explanations/titles', and erasing characters and typos in scripts does not look polished. 
With these caveats, LGTM

**Comment by kerneltime** [comment]
I should be able to re-write the esx admin cli demo in an automated manner (need to fix the cli first to report when columns are specified).
Also, should be able to get rid of "Echo" command, needs changes to spielbash.. 
@msterin if you see a typo please, pointing it out would be helpful, will rescan when I update the scripts.

**Comment by kerneltime** [comment]
merging this in for now. There are larger changes for demos coming where I will address the not echo text and admin cli

**Pull request merged**